src/main/java/core/basesyntax/model/Operation.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.model;
+
+public enum Operation {
+    BALANCE("b"),
+    SUPPLY("s"),
+    PURCHASE("p"),
+    RETURN("r");
+
+    private String code;
+
+    Operation(String code) {
```

Comment:
Don't begin method implementation with an empty line. Remove the redundant empty line at the beginning of the constructor.